### ISSUE TITLE
Some questions about the results

### ISSUE THREAD

**Issue by Pedrojj97** [sunggukcha/deeplabs#8]
Hi ,I have some questions about the result you report, because I download your pretrained weights,and evaluate on the 10k eva images of BDD100K dataset,but I  just got  about 83% miou. So I want to konw is that the result you report evaluated on the 20k test images of BDD100K Dataset?

**Comment by sunggukcha** [comment]
Hello @Pedrojj97 , would you let me know your experiment detail? The more detail, the better I can help you.

**Comment by Pedrojj97** [comment]
> Hello @Pedrojj97 , would you let me know your experiment detail? The more detail, the better I can help you.

Thanks for replying,I download your pretrained weights(resnet101) and evaluated on the 10k Validation set of BDD100K dataset，and I got about 83%(ps I don't remeber the specific values) miou, So I just want to konw the 85.33% miou you report is evaluated on  10k Validation set of BDD100K or 20k test set of BDD100K

**Comment by sunggukcha** [comment]
The reported score 85.33% miou for R101/GN16 is form **20k test set**.  You can check the test scores from https://bdd-data.berkeley.edu/ (Not sure if the test server is still alive).

**Comment by Pedrojj97** [comment]
Thank you very much😀,I get the answer I want
